WORKSPACE.bzlmod

```diff
@@ -1 +1 @@
-workspace(name = "buildifier_prebuilt")
+# Intentionally blank. Force bzlmod strict mode.
```

Comment:
guess we can delete this one now if we just never update the other one again?

Reply:
Done.